jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -226,6 +226,104 @@ def vue_update_subset(self, *args):
             self.hub.broadcast(SnackbarMessage(
                 f"Failed to update Subset: {repr(err)}", color='error', sender=self))
 
+    def vue_recenter_subset(self, *args):
+        # Composite region cannot be edited. This only works for Imviz.
+        if not self.is_editable or self.config != 'imviz':  # no-op
+            return
+
+        from photutils.aperture import ApertureStats
+        from jdaviz.core.region_translators import regions2aperture, _get_region_from_spatial_subset
+
+        try:
+            reg = _get_region_from_spatial_subset(self, self.subset_selected)
+            aperture = regions2aperture(reg)
```

Comment:
would it be useful to move this into the component itself for re-use - so `self.subset_select.aperture` or `self.subset_select.to_aperture()`?  Or would it have a place upstream in glue-astronomy?

Reply:
I think this will eventually move upstream. At least Larry or I added a few of these in that table.